lib/Wizard.php

```diff
@@ -652,8 +652,8 @@ public function testLoginName($loginName) {
 		\OCP\Util::writeLog('user_ldap', 'Wiz: testLogin '. $loginName . ', filter '. $this->access->getConnection()->ldapLoginFilter, \OCP\Util::DEBUG);
 
 		$users = $this->access->countUsersByLoginName($loginName);
-		if ($this->getLDAP()->errno($cr) !== 0) {
-			throw new \Exception($this->getLDAP()->error($cr));
+		if ($this->getLDAP()->errno($cr) !== 0 && $this->getLDAP()->errno($cr) !== '0') {
```

Comment:
We could save repetition here:
```suggestion
		$errNumber = $this->getLDAP()->errno($cr);
		if ($errNumber !== 0 && $errNumber !== '0') {
```